packages/api/scripts/cmds/db-sql.js

```diff
@@ -25,6 +25,12 @@ export async function dbSqlCmd(opts) {
   })
 
   // read all the SQL files
+  const configSql = fs.readFileSync(
+    path.join(__dirname, '../../postgres/config.sql'),
+    {
+      encoding: 'utf-8',
+    }
```

Comment:
🙏 please can we make all these a little less verbose and use `'uft8'` instead of `{ encoding: 'utf-8' }`?